account_banking_payment_export/models/payment_mode.py

```diff
@@ -43,9 +43,14 @@ def suitable_bank_types(self, cr, uid, payment_mode_id=None, context=None):
             res = [t.code for t in payment_mode.type.suitable_bank_types]
         return res
 
+    def _default_type(self):
+        return self.env.ref('account_banking_payment_export.'
+                            'manual_bank_tranfer')
+
     type = fields.Many2one(
```

Comment:
@lmignon very interesting, thanks!

Can you add a commit to the branch to try if this works?

Would ```@api.returns('payment.mode.type')``` help avoiding the ```or self.env['payment.mode.type']```